pkgs/development/tools/database/sqlboiler-crdb/default.nix

```diff
@@ -0,0 +1,25 @@
+{ buildGoModule, fetchFromGitHub, lib }:
+
+buildGoModule rec {
+  pname = "sqlboiler-crbd";
+  version = "7b35c4d19c05fdc53d1efdcc074f20ee6b56f340";
```

Comment:
```suggestion
  version = "unstable-2022-06-12";
```

Reply:
> I don't see this version tagged in the repo, is this shorthand for 'latest commit as of date x'? (a quick online search was not helpful :))

You are correct, I gave you bad review, because I read the expression from end to start.

Reply:
np, appreciate it either way, so back to the hash?

Reply:
The `version` attribute should be `unstable-{DATE}` and the `rev` attribute in `fetchFromGitHub` should be the commit hash. Use `git grep 'version = "unstable'` to see other examples.